src/components/constructor-form/constructor-form.js

```diff
@@ -0,0 +1,347 @@
+          id:editableConstructorElement._id
+        }))
+      }
+      return dispatch(constructorElementMethod({
+        constructorElement,
+        id:model._id
+      }))
+    }
+  });
+
+  const handleMaterial = (e) => {
+    setFieldValue('material', e.target.value)
+  }
+  const checkboxes = [
+    {
+      id: 'available',
+      dataCy: 'available',
+      value: values.available,
+      checked: values.available,
+      color: 'primary',
+      label: config.labels.model.show,
+      handler: () => setFieldValue('available', !values.available)
+    }
+  ];
+
+  const handleImageLoad = (e, callback) => {
+    if (e.target.files && e.target.files[0]) {
+      const reader = new FileReader();
+      reader.onload = (event) => {
+        callback(event);
+      };
+      reader.readAsDataURL(e.target.files[0]);
+    }
+  };
+
+  const handleLoadConstructorImage = (e) => {
+    handleImageLoad(e, (event) => {
+      setFieldValue('image', event.target.result);
+      setConstructorImg(event.target.result);
+    });
+  };
+
+  const textField = (inputValue, inputLabel) => (<TextField
+    data-cy={inputValue}
+    id={inputValue}
+    className={styles.textField}
+    variant='outlined'
+    label={inputLabel}
+    value={inputValue}
+    onChange={handleChange}
+    error={touched.inputValue && !!errors.inputValue}
+  />)
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <CheckboxOptions options={checkboxes} />
+        <Grid item xs={12}>
+          <Paper className={styles.constructorItemUpdate}>
+            <div>
+              <span className={styles.imageUpload}>
+                {config.labels.model.constructorPhoto}
+              </span>
+              <div className={styles.imageUploadAvatar}>
+                <ImageUploadContainer handler={handleLoadConstructorImage} />
+                {constructorImg && (
+                  <Avatar src={constructorImg} variant='rounded' className={styles.avatar}>
+                    <Image />
+                  </Avatar>
+                )}
+                {touched.image &&
+                errors.image && (
+                  <div className={styles.inputError}>
+                    {errors.image}
+                  </div>
+                )}
+              </div>
+            </div>
+            {textField(values.basePrice, config.labels.model.basePrice)}
+            {touched.basePrice && errors.basePrice && (
+              <div className={styles.inputError}>{errors.basePrice}</div>
+            )}
+            <FormControl variant='outlined' className={styles.textField}>
+              <InputLabel id='multiple-checkbox-label'>
+                {config.labels.model.material}
+              </InputLabel>
+              <Select
+                labelId='multiple-checkbox-label'
+                id='multiple-checkbox'
+                onChange={handleMaterial}
+                input={<Input />}
+                defaultValue={values.material}
```

Comment:
Please, change it to 'value' , to make this Select controlled

Reply:
Fixed